### ISSUE TITLE
RSB Ariane config incorrect dry mass for ESC-A upper stage

### ISSUE THREAD

**Issue by chuckstables** [KSP-RO/RealismOverhaul#2573]
Hi all,

It looks like there's an issue with the dry mass for the ESC-A upper stage in the Ariane RSB config file. 

The problem is that the ESA considers the interstage adapter (725 kg's apparently, but I can't find a source on this apart from the documentation in the config file) vehicle equipment bay (1300 kg's, has the structural elements needed to mount a payload to the upper stage, and contains batteries/guidance systems), and the LVA 3936 adapter (the adapter between the actual tank and the payload attachment system; two versions, one weighs 200kg's, one weighs 170 kg's).

So we shouldn't use Arianespace's dry mass figure, as we have parts for the PAF's/adapters, plus the 725 kg interstage structure that they consider "part" of the upper stage (even though it doesn't stay with the upper stage during the burn). Plus currently the upper stage dry mass figures we're using are 4.87 tonnes; even if we think that Arianespace isn't including the interstage in their dry mass figures (which it definitely seems like they are), that's still 0.33 tonnes heavier than the actual quoted figure of 4.54 tonnes (actually 0.495 tonnes heavier, as the part doesn't include the HM-7B engine, actually it'd be 0.695 tonnes heavier due to the fact the part doesn't have the LVA adapter) ). 

There is a NASA source which puts the mass of the tank itself at 1945 kg's https://www.nasa.gov/pdf/382034main_018%20-%2020090706.05.Analysis_of_Propellant_Tank_Masses.pdf). Adding the interstage adapter to that (allegedly 725 kg's, might be larger though, not sure), the HM-7B engine (165 kg's), the Vehicle Equipment Bay (1300 kg's), the 200kg LVA adapter, and the tank mass of 1945 kg's I get 4325 kg's, which is pretty damn close to their figure. The remaining  I think that should be convincing evidence that I'm correct, as it's pretty close to the 4525 figure that's quoted. 

I think the mass should be changed to be 4.525 - 0.725 - 0.165 =3.635 tonnes. We could subtract the LVA adapter's 200 kg's as well, there are two versions so it's not exactly a "fixed" part of the upper stage like the Vehicle Equipment Bay is. So it should either be 3.635 tonnes of 3.435 tonnes.

As an aside; for whatever reason ULA doesn't include the equivalent of their Vehicle Equipment Bay in their dry mass figures. Their Centaur Forward Adapter weighs 925 kg's and it fulfills the same role as the Vehicle Equipment Bay (mainly structural support), with avionics packages beneath it I believe. So it's a little inconsistent to include it in one RSB upper stage but not others, but that's just my opinion. 

If the maintainers are fine with me fixing and doing a PR just let me know and I'll do that.

**Comment by DRVeyl** [comment]
Yes, please PR.  Apologies that the GH Issue itself hasn't generated a response.  We're a bit better at sorting through PRs.
